lib/generators/solidus_shipstation/install/install_generator.rb

```diff
@@ -3,29 +3,14 @@
 module SolidusShipstation
   module Generators
     class InstallGenerator < Rails::Generators::Base
-      class_option :auto_run_migrations, type: :boolean, default: false
-
-      def add_javascripts
-        append_file 'vendor/assets/javascripts/solidus_shipstation/frontend/all.js', "//= require solidus_shipstation/frontend/solidus_shipstation\n"
-        append_file 'vendor/assets/javascripts/solidus_shipstation/backend/all.js', "//= require solidus_shipstation/backend/solidus_shipstation\n"
-      end
-
-      def add_stylesheets
-        inject_into_file 'vendor/assets/stylesheets/solidus_shipstation/frontend/all.css', " *= require solidus_shipstation/frontend/solidus_shipstation\n", before: %r{\*/}, verbose: true
-        inject_into_file 'vendor/assets/stylesheets/solidus_shipstation/backend/all.css', " *= require solidus_shipstation/backend/solidus_shipstation\n", before: %r{\*/}, verbose: true
-      end
-
-      def add_migrations
-        run 'bin/rails railties:install:migrations FROM=solidus_shipstation'
+      def self.source_paths
+        paths = superclass.source_paths
+        paths << File.expand_path('templates', __dir__)
+        paths.flatten
       end
```

Comment:
Is this any different than [this approach](https://github.com/solidusio-contrib/solidus_klaviyo/blob/master/lib/generators/solidus_tracking/install/install_generator.rb#L6), which is the standard we've just introduced in solidus_dev_support?

Reply:
No, I think it's the same. There are many ways of doing this of course but I decided to just copy what we are doing in SolidusAuthDevise, just for consistency and to be able to replace the same thing everywhere easily. I'm fine with this other way too, less code is better.